app/Wiki.php

```diff
@@ -108,6 +113,11 @@ public function settings(): \Illuminate\Database\Eloquent\Relations\HasMany
         return $this->hasMany(WikiSetting::class);
     }
 
+    public function notification(): \Illuminate\Database\Eloquent\Relations\HasMany
+    {
+        return $this->hasMany(WikiNotificationSentRecord::class);
+    }
+
```

Comment:
I think this can be deleted now?

Reply:
I guess so. I don't think we are using it anywhere now but I'm really getting confused with our models and relationships, how we expect them to look like, how we want to use them, etc. But those worries feel out of scope for this, I'll take that with me and remove this here